plugins/chat/spec/models/chat/channel_spec.rb

```diff
@@ -4,6 +4,52 @@
   fab!(:category_channel1) { Fabricate(:category_channel) }
```

Comment:
This is _super_ minor but when changing these old specs with the fabs/lets in the format `category_channel1` I also try to change them to match our more used format `category_channel_1` :)